common/src/test/java/tachyon/util/io/FileUtilsTest.java

```diff
@@ -70,4 +70,22 @@ public void changeLocalDirPermissionTests() throws IOException {
     FileUtils.changeLocalFilePermission(mTestFolder.getRoot().getAbsolutePath(), "744");
     Assert.assertTrue(tempFile.delete());
   }
+
+  @Test
+  public void moveFileTest() throws IOException {
+    File fromFile = mTestFolder.newFile("from.txt");
+    File toFile = mTestFolder.newFile("to.txt");
+    // Move a file and verify
+    FileUtils.move(fromFile, toFile);
+    Assert.assertFalse(fromFile.exists());
+    Assert.assertTrue(toFile.exists());
+  }
+
+  @Test
+  public void deleteFileTest() throws IOException {
+    File toDelete = mTestFolder.newFile("toDelete.txt");
+    // Delete a file and verify
+    FileUtils.delete(toDelete);
+    Assert.assertFalse(toDelete.exists());
+  }
```

Comment:
@apc999 looks like deleting a non-existent file doesn't result in IOException, http://docs.oracle.com/javase/7/docs/api/java/io/File.html#delete%28%29